src/integration/common/footer.js

```diff
@@ -23,5 +22,30 @@ export default () => {
       expect(brandingLinkEl.getAttribute('href')).toMatchSnapshot();
       expect(brandingImageEl).toBeInTheDocument();
     });
+
+    describe('Link', () => {
+      const footerLinks = document.querySelectorAll('footer a');
+
+      footerLinks.forEach(footerLink => {
+        const linkText = footerLink.textContent;
+        const linkUrl = footerLink.getAttribute('href');
+
+        it('should be in the document', () => {
+          expect(footerLink).toBeInTheDocument();
+        });
+
+        it('should contain text', () => {
+          expect(linkText).toBeTruthy();
+        });
+
+        it('should match text', () => {
+          expect(linkText).toMatchSnapshot();
+        });
+
+        it(`with text '${linkText}' should match url`, () => {
```

Comment:
I'm not sure how I feel about having things we are testing as part of the test title. Just doesn't seem right

Reply:
OK, I'll remove it.